syft/frameworks/torch/hook/hook.py

```diff
@@ -103,6 +105,7 @@ def __init__(
 
         # Add all the torch attributes in the syft.torch attr
         syft.torch = TorchAttributes(torch, self)
+        syft.framework = syft.torch
```

Comment:
What is the behavior if I call both Torch Hook and Tensorflow hook? Should there be a check to see if this value is already set

Reply:
I was hoping to figure out a way to make swapping between various hooks more well-defined and programmatic -- likely deserves an issue

Reply:
https://github.com/OpenMined/PySyft/issues/2530